### PR TITLE
Stabilize the unstable empty component.

### PR DESCRIPTION
**Pull request by HyopeR**
Simply put, making EmptyComponent more flexible. The following 3 parameters can cause EmptyComponent to take different styles.

```
hideMarginalsOnEmpty
hideHeaderOnEmpty
hideFooterOnEmpty
```

To solve this follow a way as HeaderComponent or FooterComponent is created. The style we give to renderEmpty or ListEmptyComponent will be one-to-one consistent.

**For example;**

```
<BigList
    data={[]}
    renderItem={() => <View />}
    itemHeight={50}
    style={{flexGrow: 1}}
    contentContainerStyle={{flexGrow: 1}}
    renderEmpty={() => {
        // It cannot fill the page. This situation has been corrected.
        return (
            <View style={{flexGrow: 1, backgroundColor: 'red'}} />
        )
    }}
/>
```

**Correction of the previously closed issue:** [https://github.com/marcocesarato/react-native-big-list/issues/83](https://github.com/marcocesarato/react-native-big-list/issues/83)

### CODE REVIEW

**Comment by HyopeR** [comment]
@marcocesarato can you check?

File: lib/BigList.jsx [hunk 1]
```diff
@@ -985,8 +990,6 @@ class BigList extends PureComponent {
           <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
         ) : null,
       contentContainerStyle: {
-        flexDirection: "row",
```
Comment (anastely) [inline lib/BigList.jsx:988]: Why you delete this 👀

Comment (anastely) [inline lib/BigList.jsx:988]: Why did you remove this? 👀
